src/test/java/com/stormnet/TestCasesAutomation.java

```diff
@@ -1,188 +1,140 @@
 package com.stormnet;
 
-import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.Configuration;
-import org.apache.commons.lang3.RandomStringUtils;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Test;
-import org.openqa.selenium.Alert;
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.openqa.selenium.remote.DesiredCapabilities;
-
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
 
-import static com.codeborne.selenide.Browsers.CHROME;
+import com.stormnet.base.BaseTest;
+import com.stormnet.page.ConfirmedOrderPage;
+import com.stormnet.page.HomePage;
+import com.stormnet.utils.Utils;
+import org.junit.jupiter.api.Test;
 import static com.codeborne.selenide.Selenide.*;
 
-public class TestCasesAutomation {
-  @BeforeAll
-  public static void setUp() {
-    Configuration.baseUrl = "http://automationpractice.com";
-    Configuration.timeout = 4005;
-    Configuration.browser = "chrome";
-    Configuration.browserSize = "1400x920";
-    Configuration.browserPosition = "200x20";
-    Configuration.headless = false;
-    Configuration.savePageSource = false;
-
-    DesiredCapabilities capabilities = new DesiredCapabilities();
-    capabilities.setBrowserName(CHROME);
-    final ChromeOptions options = new ChromeOptions();
-    capabilities.setCapability(ChromeOptions.CAPABILITY, options);
-    capabilities.setCapability("enableVNC", true);
-    Configuration.fastSetValue = true;
-    Configuration.browserCapabilities = capabilities;
-  }
+public class TestCasesAutomation extends BaseTest {
 
   @Test
   public void registration() {
-    open("/index.php");
-    $(".login").click();
-    $("#email_create").sendKeys(RandomStringUtils.randomAlphabetic(5) + "@yopmail.com");
-    $("#SubmitCreate").click();
-    $("#id_gender1").click();
-    $("#customer_firstname").sendKeys("Ivan");
-    $("#customer_lastname").sendKeys("Ivanov");
-    $("#passwd").sendKeys("12345");
-    $("#days").selectOptionByValue("1");
-    $("#months").selectOptionByValue("1");
-    $("#years").selectOptionByValue("2002");
-    $("#uniform-newsletter").click();
-    $("#uniform-optin").click();
-    $("#company").sendKeys("Kaseya");
-    $("#address1").sendKeys("701 Brickell Avenue");
-    $("#city").sendKeys("Miami");
-    $("#id_state").selectOptionByValue("9");
-    $("#postcode").sendKeys("33131");
-    $("#id_country").selectOptionByValue("21");
-    $("#other").sendKeys("Additional Information");
-    $("#phone").sendKeys("+375292929297");
-    $("#phone_mobile").sendKeys("+375333333333");
-    $("#alias").sendKeys("18, Baker street");
-    $("#submitAccount").click();
-    $(".logout[title=\"Log me out\"]").click();
+    HomePage homePage = open("/", HomePage.class);
```

Comment:
take it out in the beforeeeah method

Reply:
fixed
<img width="1680" alt="Screen Shot 2022-05-21 at 3 02 20 PM" src="https://user-images.githubusercontent.com/34116905/169761837-d2654bba-5781-43bb-aa7c-08b6bf5a71bf.png">
d